#Service to reset all encoders to zero

--- 
int64 ret
string msg
